libs/platform/src/lib/form/multi-combobox/multi-combobox/multi-combobox.component.ts

```diff
@@ -402,8 +400,6 @@ export class MultiComboboxComponent extends BaseMultiCombobox implements OnInit,
 
         // setting value, it will call setValue()
         this.value = selectedItems;
-
-        this._emitChangeEvent();
```

Comment:
EventEmitter and Form's API  are two different thing, why are you removing it from here?

Reply:
Clarified internally. 

`this.value = .. , `, calls `writeValue` and this FormAPI method already emits `selectionChange` event. so its safe to remove